corehq/apps/data_interfaces/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from tests import *
```

Comment:
Yup agreed. The crazier thing was I had to check in an empty models.py file in order to get the tests to run. Beats me why django has a hard dependency on that file being present. I think django 1.7 handles finding tests much better